aws/resource_aws_api_gateway_documentation_part_test.go

```diff
@@ -0,0 +1,243 @@
+package aws
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/apigateway"
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccAWSAPIGatewayDocumentationPart_basic(t *testing.T) {
+	var conf apigateway.DocumentationPart
+
+	rString := acctest.RandString(8)
+	apiName := fmt.Sprintf("tf_acc_api_%s", rString)
```

Comment:
Generic question: should we try to include the test name (e.g. `t.Name()`) instead of the fairly diverse naming we have across resources for acceptance tests? I'm guessing it really depends on naming restrictions for some resources (max length, no underscores, etc).

Reply:
I agree we should make it test-specific, but `t.Name()` is a bit tricky for the reasons you mentioned, so I'll come up with custom, static name.